qiskit/qasm/node/binaryoperator.py

```diff
@@ -48,5 +48,5 @@ def operation(self):
             raise NodeException(f"internal error: undefined operator '{self.value}'") from ex
 
     def qasm(self):
-        """Return the QASM representation."""
+        """Return the OpenQASM2 representation."""
```

Comment:
This might be a PITA, but doesn't it make sense to choose uniform language for describing the `qasm` methods? OTOH the `qasm` methods may have differing semantics.

Reply:
https://github.com/Qiskit/qiskit-terra/blob/dcd7beafb190c7d8646559795eafbbad9debb90e/qiskit/qasm/node/binaryoperator.py#L13

Here "OPENQASM" should be "OpenQASM"

Reply:
There are about 100 more occurrences of "OPENQASM" in qiskit-terra/qiskit/qasm/node

Reply:
We're about to deprecate the entire `qiskit.qasm` module, tbh none of this should ever really have been considered public (it's certainly not documented), so I think we're ok to skip this.